code/game/jobs/job/assistant_vr.dm

```diff
@@ -22,7 +22,10 @@
 					  "Lab Assistant" = /datum/alt_title/intern_sci,
 					  "Security Cadet" = /datum/alt_title/intern_sec,
 					  "Jr. Cargo Tech" = /datum/alt_title/intern_crg,
-					  "Server" = /datum/alt_title/server)
+					  "Server" = /datum/alt_title/server,
+					  //CHOMPSTATION edit start.
+					  "Belly Filler" = /datum/alt_title/prey)
```

Comment:
Might as well just give this single line a `// CHOMPSTATION Edit`, instead of two different comments.

Reply:
Hnnnnnnh, I forget if there's a problem when you comment stuff inside of lists. LETS TRY ANYWAY.